server/helpers/passportCallback.js

```diff
@@ -1,4 +1,4 @@
-import { userResponse } from "./userResponse";
+import { userResponse } from "./users";
```

Comment:
`userResponses` be consistent with the file you are referencing

Reply:
I decided to put the `userResponse` and `userProfileResponse` in the same file called `users`, that way if anyone wants to define their own response they can put it there as well...
I thought what what you meant by the initial comment

Reply:
Should I put them in seperate files

Reply:
yea sure, what I meant was change to `userResponses` because there are many of them

Reply:
Oh, okay... I'll fix it now